src/LendingPool.sol

```diff
@@ -162,6 +164,14 @@ contract LendingPool is Guardian, TrustedCreditor, DebtToken, InterestRateModule
         treasury = treasury_;
     }
 
+    /**
+     * @notice Sets the new origination fee
+     * @param originationFee_ The new origination fee
+     */
+    function setOriginationFee(uint16 originationFee_) external onlyOwner {
```

Comment:
Hmmm now we can set it to 6,5x the debt, looks abusable

Reply:
changed it to uint8, max 2.55%